### PR TITLE
fix: table pagination

### LINKED ISSUE #45: o-table pagination issue
Hey guys, I'm currently trying to integrate Oruga into my app that has been using Buefy. 

I'm using Oruga `0.5.2`  and theme-bulma `0.2.1`. 

I'm having an issue with the appearance of the pagination section when I use the Oruga Table Component `o-table`. 

When I compare it to Buefy's pagination for `b-table`, I notice that all the level classes aren't being applied. For example: `level top`, `level-right`, `level-left` and so on. 

As a temporary fix, I manually wrote the scss for the level classes to be applied to the pagination divs. 

I've included the styles below...

I think a more permanent solution will have to be an update to both theme-bulma and oruga. Oruga needs to have the customizations for those divs so theme-bulma can set the configuration for the level classes. Right now, the only customization for this is `paginationWrapperClass`, and theme-bulma sets this to empty when it should at least make it the `level` class.

```
// Temp solution to getting the pagination to look right.
.b-table.table-wrapper {
  margin-bottom: 0;

  .pagination-link {
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  + div {
    padding: 1.125rem 1.5rem;
    margin: 0;
    border-top: rgba(24, 28, 33, 0.06);
    background: #f5f5f5;
    display: flex;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-box-pack: justify;
    -ms-flex-pack: justify;
    justify-content: space-between;

    div {
      display: flex;
      flex-basis: auto;
      -webkit-box-flex: 0;
      flex-grow: 0;
      flex-shrink: 0;
      -webkit-box-align: center;
      -ms-flex-align: center;
      align-items: center;
    }
    div:nth-child(1) {
      -webkit-box-pack: start;
      -ms-flex-pack: start;
      justify-content: flex-start;
    }
    div:nth-child(2) {
      -webkit-box-pack: end;
      -ms-flex-pack: end;
      justify-content: flex-end;

      div {
        -webkit-box-align: center;
        -ms-flex-align: center;
        align-items: center;
        display: -webkit-box;
        display: -ms-flexbox;
        display: flex;
        flex-basis: auto;
        -webkit-box-flex: 0;
        flex-grow: 0;
        flex-shrink: 0;
        -webkit-box-pack: center;
        -ms-flex-pack: center;
        justify-content: center;
      }
    }
  }
}
```

### PR DESCRIPTION
**Pull request by MaTTze**
fixes #45

### CODE REVIEW

File: src/assets/scss/components/_table.scss [hunk 1]
```diff
@@ -82,7 +82,7 @@ $table-sticky-header-height: 300px !default;
     }
 }
 
-.b-table {
+.o-table {
```
Comment (jtommy) [inline src/assets/scss/components/_table.scss:85]: Why???

**Review by jtommy — CHANGES_REQUESTED** [review]

File: src/assets/scss/components/_table.scss (hunk 1 above)
Comment (MaTTze) [reply src/assets/scss/components/_table.scss:85]: I thought it was an typo. Should it stay `b-table`?

Comment (jtommy) [reply src/assets/scss/components/_table.scss:85]: Yes!!

File: src/assets/scss/components/_table.scss [hunk 2]
```diff
@@ -265,17 +265,44 @@ $table-sticky-header-height: 300px !default;
...
+    }
     // .table-wrapper {
     &.table-wrapper {
         position: relative;
         &.has-sticky-header {
             height: $table-sticky-header-height;
             overflow-y: auto;
-            &.has-mobile-cards {
+            &.is-mobile {
                 @include mobile {
```
Comment (jtommy) [inline src/assets/scss/components/_table.scss:306]: is-mobile too, the table uses has-mobile-cards

**Review by jtommy — CHANGES_REQUESTED** [review]

File: src/assets/scss/components/_table.scss (hunk 2 above)
Comment (jtommy) [reply src/assets/scss/components/_table.scss:306]: In bulma.js there is the other class

Comment (jtommy) [reply src/assets/scss/components/_table.scss:306]: Yes, then you have to change it on bulma.ts

Comment (MaTTze) [reply src/assets/scss/components/_table.scss:306]: I did 😉

**Review by jtommy — APPROVED** [review]

**Pull request merged**